yt/frontends/enzo_e/data_structures.py

```diff
@@ -406,9 +406,12 @@ def _parse_parameter_file(self):
         ablock = fh[list(fh.keys())[0]]
         self.current_time = ablock.attrs["time"][0]
         self.parameters["current_cycle"] = ablock.attrs["cycle"][0]
-        gsi = ablock.attrs["enzo_GridStartIndex"]
+        gsi = ablock.attrs["enzo_GridStartIndex"]  # <- always has 3 elements
```

Comment:
Instead of a comment, you could add an `assert` here, which would be less likely to rot silently

Reply:
Good point! Fixed